stubs/annoy/annoy/annoylib.pyi

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+class Annoy:
+    f: Any
+    def __init__(self, *args, **kwargs) -> None: ...
+    def add_item(self, *args, **kwargs) -> Any: ...
+    def build(self, *args, **kwargs) -> Any: ...
+    def get_distance(self, *args, **kwargs) -> Any: ...
+    def get_item_vector(self, *args, **kwargs) -> Any: ...
+    def get_n_items(self, *args, **kwargs) -> Any: ...
+    def get_n_trees(self, *args, **kwargs) -> Any: ...
+    def get_nns_by_item(self, *args, **kwargs) -> Any: ...
+    def get_nns_by_vector(self, *args, **kwargs) -> Any: ...
+    def load(self, *args, **kwargs) -> Any: ...
+    def on_disk_build(self, *args, **kwargs) -> Any: ...
+    def save(self, *args, **kwargs) -> Any: ...
+    def set_seed(self, *args, **kwargs) -> Any: ...
+    def unbuild(self, *args, **kwargs) -> Any: ...
+    def unload(self, *args, **kwargs) -> Any: ...
+    def verbose(self, *args, **kwargs) -> Any: ...
```

Comment:
Should these really all be `-> Any`, or should we just leave the return types unannotated?